app/controllers/hyrax/admin/admin_sets_controller.rb

```diff
@@ -113,14 +114,6 @@ def form
         @form ||= form_class.new(@admin_set, current_ability, repository)
       end
 
-      # Overrides the parent implementation so that the returned search builder
-      #  searches for edit access
-      # Instantiates the search builder that builds a query for a single item
-      # this is useful in the show view.
-      def single_item_search_builder
-        single_item_search_builder_class.new(self, :edit).with(params.except(:q, :page))
-      end
-
```

Comment:
Remove the restriction that the show page be limited to those with `:edit` access.